src/components/Popover/LazyTippy.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import Tippy from '@tippyjs/react';
+
+// Will only render the `content` or `render` elements if the tippy is mounted to the DOM.
+// Replace <Tippy /> with <LazyTippy /> component and it should work the same.
+
+/* eslint react/prop-types: 0 */
```

Comment:
I really don't fee like this component should have prop types as that could create issues later on if Tippy updates it's proptypes, these could get out of sync.

Reply:
There are still advantages for this component to explicitly declare its dependencies and document them. We can do it in a follow up though